projects/plugins/jetpack/extensions/blocks/ai-assistant/transforms/index.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { createBlock } from '@wordpress/blocks';
+/**
+ * Internal dependencies
+ */
+import { blockName } from '..';
+import { EXTENDED_BLOCKS, isPossibleToExtendBlock } from '../extensions/ai-assistant';
+import { PromptItemProps } from '../lib/prompt';
+
+const transformFromCoreEmbed = {
+	type: 'block',
+	blocks: EXTENDED_BLOCKS,
+	isMatch: () => isPossibleToExtendBlock(),
+	transform: ( { content } ) => {
+		const messages: Array< PromptItemProps > = [
+			{
+				role: 'user',
+				content,
+			},
+		];
+
+		return createBlock( blockName, { content, messages } );
+	},
+};
+
+const from = [ transformFromCoreEmbed ];
```

Comment:
```suggestion
const from = [ transformFromCore ];
```

In case we decide to rename it as suggested above.